lib/vanagon/platform/dsl.rb

```diff
@@ -97,6 +100,27 @@ def package_type(pkg_type)
         @platform.package_type = pkg_type
       end
 
+      # Set the path to find for the platform
+      #
+      # @param find_cmd [String] Full path to the find command for the platform
+      def find(find_cmd)
```

Comment:
No.... only if you want to set `plat.find= /usr/bin/find` in the platform config. This goes with the current practice of setting variables, so the user specified `plat.find "/usr/bin/find"`